cli/azd/internal/appdetect/java.go

```diff
@@ -236,6 +239,12 @@ func detectDependencies(mavenProject *mavenProject, project *Project) (*Project,
 				UseKafka: true,
 			})
 		}
+
+		if dep.GroupId == "com.azure.spring" && dep.ArtifactId == "spring-cloud-azure-starter" {
+			project.AzureDeps = append(project.AzureDeps, SpringCloudAzureDep{
+				Version: dep.Version,
```

Comment:
Will this version can be got if the pom file is using maven dependency or bom?

Reply:
The answer is no. 
Currently we only determine if it's used or not, not care its version. So this field is not used yet, I will remove it first.